src/esri-loader.ts

```diff
@@ -72,7 +80,9 @@ export const utils = {
 export function getScript() {
   return document.querySelector('script[data-esri-loader]') as HTMLScriptElement;
 }
-
+export function getCss() {
+  return document.querySelector('style[data-esri-loader]') as HTMLStyleElement;
```

Comment:
I think this would actually be a [HTMLLinkElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLinkElement)